Move @types/webdriverio from devDependencies to dependencies

The Spectron declaration file [spectron.d.ts](https://github.com/electron-userland/spectron/blob/master/lib/spectron.d.ts) depends on types from `@types/webdriverio`.

In a Typescript project depending on spectron, typescript throws the error
```
node_modules/spectron/lib/spectron.d.ts(9,34): error TS7016: Could not find a declaration file for module 'webdriverio'. '-------/node_modules/webdriverio/build/index.js' implicitly has an 'any' type.
[typescript]   Try `npm install @types/webdriverio` if it exists or add a new declaration (.d.ts) file containing `declare module 'webdriverio';`
```
This is because `@types/webdriverio` is defined as devDependency and therefore not installed in projects depending on `spectron`.

A simple workaround is to install `@types/webdriverio` yourself. This is not a permament solution because for every new version of `spectron` you have to check if your defined version of `@types/webdriverio` still fits.

--- a/package.json
+++ b/package.json
@@ -24,6 +24,7 @@
   "author": "Kevin Sawicki",
   "license": "MIT",
   "dependencies": {
+    "@types/webdriverio": "^4.8.0",
     "dev-null": "^0.1.1",
     "electron-chromedriver": "^7.0.0",
     "request": "^2.87.0",
@@ -31,7 +32,6 @@
     "webdriverio": "^4.13.0"
   },
   "devDependencies": {
-    "@types/webdriverio": "^4.8.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "chai-roughly": "^1.0.0",

